resources/bundles/org.eclipse.core.filesystem/src/org/eclipse/core/internal/filesystem/local/LocalFile.java

```diff
@@ -447,6 +447,7 @@ private boolean isSameFile(File source, File destination) throws IOException {
 		}
 	}
 
+	/** like {@link #readAllBytes(int, IProgressMonitor)} */
```

Comment:
I am not used to @inheritDoc, but it seems unrelated, since i do not try to inherit the documentation from the overriden. Also see https://www.cmos.blog/sinn-unsinn-von-inheritdoc/
Can you please suggest a concrete rewording? The idea is that all those methods are very similar and just return another Datatype.

Reply:
Please avoid using inheritdoc, it makes comments much harder to explore both in the Eclipse IDE as well as in generated Javadoc (because both the Javadoc view as well as the doclet generate better information _without_ that tag).

Reply:
ok, changed to @see